src/libraries/Native/Unix/System.Security.Cryptography.Native.Android/pal_rsa.c

```diff
@@ -321,17 +318,17 @@ PALEXPORT int32_t CryptoNative_GetRsaParameters(RSA* rsa,
     return CheckJNIExceptions(env) ? FAIL : SUCCESS;
 }
 
-jobject BigNumFromBinary(JNIEnv* env, uint8_t* bytes, int32_t len)
+static jobject BigNumFromBinary(JNIEnv* env, uint8_t* bytes, int32_t len)
```

Comment:
The difference between this and AndroidCryptoNative_BigNumFromBinary is very small.  Can this one be deleted?